<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyEditLSY</class>
 <widget class="QMainWindow" name="MyEditLSY">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyEditLSY</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>211</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_New"/>
    <addaction name="action_Open"/>
    <addaction name="action_Save"/>
    <addaction name="action_Save_As"/>
    <addaction name="separator"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_Option">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="action_Cut"/>
    <addaction name="action_Copy"/>
    <addaction name="action_Past"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About"/>
    <addaction name="action_About_QT"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Option"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_New"/>
   <addaction name="action_Open"/>
   <addaction name="action_Save"/>
   <addaction name="separator"/>
   <addaction name="action_Cut"/>
   <addaction name="action_Copy"/>
   <addaction name="action_Past"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_New">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/new.png</normaloff>:/images/new.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="toolTip">
    <string>Create new file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/open.png</normaloff>:/images/open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="toolTip">
    <string>Open new file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/save.png</normaloff>:/images/save.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="toolTip">
    <string>Save a file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_Save_As">
   <property name="text">
    <string>&amp;Save As</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="action_Cut">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/cut.png</normaloff>:/images/cut.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Cut</string>
   </property>
   <property name="toolTip">
    <string>Cut  a fdata</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="action_Copy">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/copy.png</normaloff>:/images/copy.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Copy</string>
   </property>
   <property name="toolTip">
    <string>Copy a text</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="action_Past">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/paste.png</normaloff>:/images/paste.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Paste</string>
   </property>
   <property name="toolTip">
    <string>Paste a text</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="action_About_QT">
   <property name="text">
    <string>&amp;About QT</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>action_Cut</sender>
   <signal>triggered()</signal>
   <receiver>textEdit</receiver>
   <slot>cut()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>185</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Copy</sender>
   <signal>triggered()</signal>
   <receiver>textEdit</receiver>
   <slot>copy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>185</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Past</sender>
   <signal>triggered()</signal>
   <receiver>textEdit</receiver>
   <slot>paste()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>185</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>textEdit</sender>
   <signal>copyAvailable(bool)</signal>
   <receiver>action_Cut</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>185</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>185</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>textEdit</sender>
   <signal>copyAvailable(bool)</signal>
   <receiver>action_Copy</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>185</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>185</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
